R/config.R

```diff
@@ -25,11 +25,10 @@
     set_config_value(name, values[[name]])
   })
 
-  logger::log_info("Config values initialized.", namespace = "pharmapkgs")
-  logger::log_debug(
-    "{jsonlite::toJSON(as.list(.config), auto_unbox = TRUE, pretty = TRUE)}",
-    namespace = "pharmapkgs"
-  )
+  packageStartupMessage("Pharmapkgs: config values loaded")
```

Comment:
This is not a message from the Start up. I would recommend the more idiomatic:

```suggestion
  message("Pharmapkgs: config values loaded")
```

A similar change would be needed on multiple places of the package

Reply:
These messages are actually from the startup - because `.init_config_values()` and `.setup_logger()` are both called inside the `.onLoad` hook.

In fact, I am using `packageStartupMessage` here to avoid the following R CMD check NOTE:

```
N  checking whether startup messages can be suppressed ...
   Pharmapkgs log level:DEBUG
   Pharmapkgs: config values loaded
   {
     "excluded_riskmetric_assessments": ["assess_covr_coverage", "assess_r_cmd_check"],
     "local_packages": "/tmp/Rtmpop2y4c/filea3cd64a773c9/pharmapkgs.Rcheck/pharmapkgs/repos/src/contrib/PACKAGES",
     "project_path": "/tmp/Rtmpop2y4c/filea3cd64a773c9/pharmapkgs.Rcheck/00_pkg_src/pharmapkgs",
     "local_base": "/tmp/Rtmpop2y4c/filea3cd64a773c9/pharmapkgs.Rcheck/pharmapkgs/repos",
     "remote_base": "https://cloud.r-project.org/",
     "limit": 5
   }
   
   It looks like this package (or a package it requires) has a startup
   message which cannot be suppressed: see ?packageStartupMessage.
```

Reply:
Oh,  maybe I didn't get that message because I was loading it via devtools instead of a normal installation. Thanks for clarifying!